spec/features/user_visits_book_show_page_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe 'a user visits the book show page' do 
+   before(:each) do 
+    @book = create(:book)
+    @user = create(:user)
+    @book.reviews.create!(body: 'a review', rating: 1, book_id: @book.id, user_id: @user.id)
```

Comment:
Since you are creating this record on a book, you don't need include book_id on line 7.